backend/app/services/chat/messages/converter.py

```
@@ -0,0 +1,230 @@
+        if image_base64:
+            # Decode, compress, then re-encode
+            try:
+                image_data = base64.b64decode(image_base64)
+                if len(image_data) > MAX_IMAGE_SIZE_BYTES:
+                    compressed_data = MessageConverter._compress_image(
+                        image_data, mime_type
+                    )
+                    image_base64 = base64.b64encode(compressed_data).decode("utf-8")
+            except Exception:
+                # If decoding/compression fails, proceed with original (might fail later)
+                pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)